Applications/WebPortal/WebApp/library/source/pages/group_page/group_info_page/group_info_page.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { AccountEntry, DisplaySize, PageWrapper } from '../../..';
+import { GroupMemberEntry } from './group_member_entry';
+
+interface Properties {
+
+  /** The size of the viewport. */
+  displaySize: DisplaySize;
+
+  /** The accounts that belong to the group. */
+  group: AccountEntry[];
+}
+
+/** A page that displays information about a group and its members. */
+export class GroupInfoPage extends React.Component<Properties> {
+
+  public render(): JSX.Element {
+    const {displaySize, group} = this.props;
```

Comment:
Don't destructure, for the blackjack/sudoku we'll allow it, and if this were plain JS I can see some advantages to doing it. But for TypeScript I just don't see any benefits.

Reply:
I like it because it makes the code a little bit more readable and sometimes lines can get a little long.